local/bin/py/build/configurations/pull_config.yaml

```diff
@@ -1046,8 +1046,8 @@
       options:
         dest_path: "/developers/"
         file_name: "datadog_apps.md"
-        # front_matters:
-        #   dependencies: ["https://github.com/DataDog/apps/blob/master/docs/en/gettiing-started.md"]
+        front_matters:
+          dependencies: ["https://github.com/DataDog/apps/blob/master/docs/en/gettiing-started.md"]
```

Comment:
hey @renteria1010 looks like theres a typo in both files here `gettiing-started.md`